### PR TITLE
Fixed Controller command not updating config routing file

### LINKED ISSUE #284: [DX] Potential missing annotation route import
I _believe_ there is a bug when using `doctrine:generate:crud` with the annotation format. Specifically, the routing import (to import the `Controller` directory with `type: annotation`) is not added. I believe you can see this here: https://github.com/sensiolabs/SensioGeneratorBundle/blob/master/Command/GenerateDoctrineCrudCommand.php#L122

My guess is that the thought is that if you originally generated the bundle with the annotation config format, then you'll already have this import. We should do a little more work to make sure that the controller _is_ in fact being imported in routing. Without it, the CRUD seems to not work at first.

Thanks!

### PR DESCRIPTION
**Pull request by HeahDude**
Closes #284.

### CODE REVIEW

File: Command/GenerateControllerCommand.php [hunk 1]
```diff
@@ -333,4 +346,19 @@ protected function createGenerator()
     {
         return new ControllerGenerator($this->getContainer()->get('filesystem'));
     }
+
+    /**
+     * @param string $bundleName
+     * @param string $controller
+     */
+    protected function tryUpdateAnnotationRouting($bundleName, $controller)
```
Comment (fabpot) [inline Command/GenerateControllerCommand.php]: should be private, and phpdoc can be removed.

Comment (HeahDude) [inline Command/GenerateControllerCommand.php]: Done.

File: Command/GenerateControllerCommand.php [hunk 2]
```diff
@@ -333,4 +346,15 @@ protected function createGenerator()
     {
         return new ControllerGenerator($this->getContainer()->get('filesystem'));
     }
+
+    private function tryUpdateAnnotationRouting($bundleName, $controller)
```
Comment (adrienlucas) [inline Command/GenerateControllerCommand.php]: Maybe this should go in `Sensio\Bundle\GeneratorBundle\Generator\ControllerGenerator::generateRouting()` ?

Comment (HeahDude) [inline Command/GenerateControllerCommand.php]: It seems that generators do not use manipulators, only commands uses both.

**Comment by HeahDude** [comment]
ping @fabpot

File: Command/GenerateControllerCommand.php (hunk 2 above)
Comment (adrienlucas) [inline Command/GenerateControllerCommand.php]: Fair enough ! (sorry for the latency)

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Pull request merged**